open-company/open-company-web
# Activity move

Card: https://trello.com/c/6IAm5ipz

### Review after #575 

To test:
- go to AP
- click more menu of a post
- [x] do you see the move icon? Good
- click move
- pick a new board
- click Apply
- [x] did the post change board in the AP list? Good
- go to a single section
- open the ellipses menu of a post
- click Move
- select another board
- click Apply
- [x] did the post disappear from the current section? Good
- [x] did the post never flickered or appeared again after disappearing? Good
- switch to the board you moved the post to
- [x] do you see the post there w/o refreshing? Good
